Guldier/settings.py

```diff
@@ -19,6 +21,7 @@
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+environ.Env.read_env(BASE_DIR / '.env')
```

Comment:
there is a repetition of this line down at 36, if not necessary please delete one

Reply:
Right, I will remove this duplication.